.github/workflows/windows.yml

```diff
@@ -1,71 +1,67 @@
 ---
-name: windows-latest
+name: Windows
 
 on: [push, pull_request]
+
 env:
   ARCH: "x64"
-  VC: "C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Enterprise\\VC\\Auxiliary\\Build\\vcvarsall.bat"
+  VC2019: "C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Enterprise\\VC\\Auxiliary\\Build\\vcvarsall.bat"
+  VC2017: "C:\\Program Files (x86)\\Microsoft Visual Studio\\2017\\Enterprise\\VC\\Auxiliary\\Build\\vcvarsall.bat"
+
 jobs:
-  compile: # name of the job. Jobs run in parallel unless specified otherwise.
-    name: compile
-    runs-on: windows-latest
+  test:
+    name: Compile and test planner
+    timeout-minutes: 60
+    runs-on: ${{ matrix.windows-version }}
     strategy:
       matrix:
-        python-version: ['3.6']
-        build-version: ["release", "debug"]
-    steps: # each - is a new sequentially run step
-      - uses: actions/checkout@master
+        windows-version: [windows-2016, windows-2019]
+        python-version: [3.6]
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@master
 
-      - name: Setup python
-        uses: actions/setup-python@v1
+      - name: Install python
+        uses: actions/setup-python@master
         with:
           python-version: ${{ matrix.python-version }}
-          architecture: ${{ env.ARCH }}
 
-      - name: compile
+      - name: Compile planner
         shell: cmd
+        env:
+          CC: cl
+          CXX: cl
```

Comment:
Shall we set these env vars globally for the workflow?

Reply:
issue1005 will change this file completely; not worth for this issue.